vue/components/ui/atoms/dropdown/dropdown.vue

```diff
@@ -223,6 +223,9 @@ export const Dropdown = {
         onMouseleave(index) {
             this.dehighlight(index);
         },
+        onSlideAfterLeave() {
+            this.$emit("animation:ended");

```

Comment:
This only happens when closing, right? If that's the case, suffix it with `close` .

Reply:
Yes, it triggers after the close animation.